### PR TITLE
added webpack

### PR DESCRIPTION
**Pull request by dannycoates**
This is just a start at webpack and transpiling to es2015. 

@pdehaan I excluded the "env" preset for now for simplicity. We should revisit it along with removing the full polyfill.min.js script from the template in a follow up issue.

Bonus: fixes Edge (I think)

### CODE REVIEW

File: frontend/src/links.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 let links = [];
 
 document.addEventListener('DOMContentLoaded', function() {
-  links = document.querySelectorAll('a:not([target])');
+  links = Array.from(document.querySelectorAll('a:not([target])'));
```
Comment (dannycoates) [inline frontend/src/links.js]: Edge doesn't have `forEach` on `NodeList`

File: frontend/src/metrics.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const testPilotGA = require('testpilot-ga');
+const testPilotGA = require('testpilot-ga/src/TestPilotGA');
```
Comment (dannycoates) [inline frontend/src/metrics.js]: This is a bit hacky but it lets us transpile the lib how we want to

File: package.json [hunk 3]
```diff
@@ -17,7 +17,12 @@
   },
   "devDependencies": {
     "asmcrypto.js": "0.0.11",
+    "babel-core": "^6.25.0",
+    "babel-loader": "^7.1.1",
+    "babel-plugin-add-module-exports": "^0.2.1",
     "babel-polyfill": "^6.23.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
     "browserify": "^14.4.0",
```
Comment (dannycoates) [inline package.json]: we should follow up to remove this, but i didn't want to mess with `test--browser` yet

File: frontend/src/metrics.js (hunk 2 above)
Comment (pdehaan) [inline frontend/src/metrics.js]: It technically _works_, but only due to a bug in https://github.com/mozilla/testpilot-ga/pull/13.
Not sure how you want to address. We'd either have to re-compile the **testpilot-ga** dist files, or kill my upstream PR and remove the bad `files` in **testpilot-ga** repo.

Comment (dannycoates) [inline frontend/src/metrics.js]: we can cross that bridge when we get there ;)

Comment (pdehaan) [inline frontend/src/metrics.js]: Totally, just wanted to point out that due to the fact of loose dependency versions (`"testpilot-ga": "^0.3.0"`), semver, and upstream repos, that could technically break at any point in the future when @chuckharmston publishes a new version of **testpilot-ga**.

**Pull request merged**